src/logic/safe/store/reducer/gatewayTransactions.ts

```diff
@@ -119,7 +118,7 @@ export const gatewayTransactionsReducer = handleActions<GatewayTransactionsState
       // If there's no "Queued" label, put all the items into the "next" group.
       let nextItems = values
       let queuedItems = values.slice(0, 0)
-      const qLabelIndex = values.findIndex((item) => isLabel(item) && item.label === LabelValue.Queued)
+      const qLabelIndex = values.findIndex((item) => isLabel(item) && item.label === 'Queued')
```

Comment:
Why change this?

Reply:
For some reason I am getting a `Cannot read properties of undefined (reading 'Queued')` error on page reload so `LabelValue` is undefined. Already tried reinstalling dependencies without success.

Reply:
Can you check which version of the CGW SDK you have in your node_modules? This enum was added in this version: https://github.com/gnosis/safe-react-gateway-sdk/releases/tag/v2.10.2

Reply:
It works again now 🤔 Will add it back